### PR TITLE
Updated readme.md to reflect new repository location

### PR DESCRIPTION
**Pull request by dgallinari**
The link to wiki was broken, so I've changed it to point to current repo location.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ Just use "hm conf" and "hm build" to compile (./hm.sh on
 Linux and Mac).
 
 For detailed compile instructions:
-http://synergy-project.org/wiki/Compiling
+https://github.com/synergy/synergy/wiki/Compiling
```
Comment (XinyuHou) [inline README.md]: Why do you rename this file to README.md?

Comment (dgallinari) [inline README.md]: I've done this in order to take advantage of Markdown markup language. So it will be more readable on GitHub page. :smile:

Comment (XinyuHou) [inline README.md]: Do you mean with .md, the file shows in a different way on Github? Could you paste a screenshot?

Comment (dgallinari) [inline README.md]: Yep. In a minute.

Comment (dgallinari) [inline README.md]: You can see the differences in the repos:
https://github.com/symless/synergy - without .md
https://github.com/dgallinari/synergy - using .md

Also, see images below:
## Before
screenshot from 2016-05-12 12 00 46

## After
screenshot from 2016-05-12 12 00 59

Comment (dgallinari) [inline README.md]: And this is just the beginning... Using markdown, we could make a pretty readme on GitHub main page, highlighting important sections, using references, lists, bold font, italic font, etc..

**Comment by nlyan** [comment]
This is now fixed. The markdown readme will be beefed up when it actually gets rewritten.

**Pull request closed without merging**